dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/Constants.java

```diff
@@ -19,8 +19,8 @@
 
 public interface Constants {
     int OPEN_DELAY = 100;
-    int CLOSE_DELAY = 0;
-    int DURATION = 2000;
+    int CLOSE_DELAY = 250;
```

Comment:
Could you explain what the close delay is?

Reply:
Close delay is the delay between detecting the mouse has left the tooltip and the tooltip disappearing. With it set to 0 it felt strange because even a little bit of mouse movement would dismiss the tooltip.